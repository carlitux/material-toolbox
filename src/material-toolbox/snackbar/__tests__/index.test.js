import React from 'react';
import { shallow } from 'enzyme';
import Snackbar from '../index';

describe('component::Snackbar', () => {
  it('Should match snapshoot', () => {
    const component = shallow(<Snackbar message="Snackbar test" />);
    expect(component).toMatchSnapshot();
  });

  it('Should match snapshoot multiline', () => {
    const component = shallow(<Snackbar message="Snackbar test" multiline />);
    expect(component).toMatchSnapshot();
  });

  it('Should match snapshoot actionOnBottom', () => {
    const component = shallow(
      <Snackbar message="Snackbar test" multiline actionOnBottom />,
    );
    expect(component).toMatchSnapshot();
  });

  it('Should match snapshoot align', () => {
    const component = shallow(<Snackbar message="Snackbar test" align />);
    expect(component).toMatchSnapshot();
  });
});
