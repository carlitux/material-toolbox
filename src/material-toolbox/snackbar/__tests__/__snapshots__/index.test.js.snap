// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component::Snackbar Should match snapshoot 1`] = `
<div
  aria-atomic="true"
  aria-hidden={true}
  aria-live="assertive"
  className="mdc-snackbar"
>
  <div
    className="mdc-snackbar__text"
  />
  <div
    className="mdc-snackbar__action-wrapper"
  >
    <button
      aria-hidden={true}
      className="mdc-snackbar__action-button"
      type="button"
    />
  </div>
</div>
`;

exports[`component::Snackbar Should match snapshoot actionOnBottom 1`] = `
<div
  aria-atomic="true"
  aria-hidden={true}
  aria-live="assertive"
  className="mdc-snackbar"
>
  <div
    className="mdc-snackbar__text"
  />
  <div
    className="mdc-snackbar__action-wrapper"
  >
    <button
      aria-hidden={true}
      className="mdc-snackbar__action-button"
      type="button"
    />
  </div>
</div>
`;

exports[`component::Snackbar Should match snapshoot align 1`] = `
<div
  aria-atomic="true"
  aria-hidden={true}
  aria-live="assertive"
  className="mdc-snackbar mdc-snackbar--align-start"
>
  <div
    className="mdc-snackbar__text"
  />
  <div
    className="mdc-snackbar__action-wrapper"
  >
    <button
      aria-hidden={true}
      className="mdc-snackbar__action-button"
      type="button"
    />
  </div>
</div>
`;

exports[`component::Snackbar Should match snapshoot multiline 1`] = `
<div
  aria-atomic="true"
  aria-hidden={true}
  aria-live="assertive"
  className="mdc-snackbar"
>
  <div
    className="mdc-snackbar__text"
  />
  <div
    className="mdc-snackbar__action-wrapper"
  >
    <button
      aria-hidden={true}
      className="mdc-snackbar__action-button"
      type="button"
    />
  </div>
</div>
`;
